app/controllers/api/v1/volunteers_controller.rb

```diff
@@ -0,0 +1,57 @@
+require 'net/http'
+module Api
+  module V1
+    class VolunteersController < ApplicationController
+      include Response
```

Comment:
You don't need to include response and exception handler. They are already inherited from the application controller as a result of the line above